creator.cpp

```diff
@@ -38,6 +39,7 @@
 #include <QDesktopServices>
 #include <QMimeData>
 #include <QProcess>
+#include <QRegExp>
```

Comment:
not clear why this include is needed

Reply:
Needed to include from `core5compat`

Reply:
If we could replace the usage with the QRegularExpression equivalent this could be removed.

Reply:
Replaced with QT6 equivalent.